tuftool/src/main.rs

```diff
@@ -116,29 +117,39 @@ where
     .context(error::FileParseJsonSnafu { path })
 }
 
-async fn write_file<T>(path: &Path, json: &T) -> Result<()>
+async fn write_file<T>(path: &Path, json: T) -> Result<()>
 where
-    T: serde::Serialize,
+    T: serde::Serialize + Send + Sync + 'static,
 {
-    // Use `tempfile::NamedTempFile::persist` to perform an atomic file write.
-    let parent = path.parent().context(error::PathParentSnafu { path })?;
-    let file =
-        NamedTempFile::new_in(parent).context(error::FileTempCreateSnafu { path: parent })?;
+    // These will be moved into a thread to prevent blocking.
+    let parent = path
```

Comment:
It's unclear what "these" is referring to. Write file operations? Maybe just leave this comment out.